MedallionShell.Tests/MedallionShell.Tests.csproj

```diff
@@ -34,5 +35,14 @@
     <Reference Include="System.Management" />
     <Reference Include="System.ServiceModel" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.2'">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces">
+      <Version>8.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Linq.Async">
```

Comment:
We don’t want the system.Lina.async dependency

Reply:
@madelson Please see https://github.com/madelson/MedallionShell/pull/109#discussion_r1515380837

Reply:
Fixed